cpp/benchmarks/search/search_benchmark.cu

```diff
@@ -153,3 +138,53 @@ BENCHMARK_REGISTER_F(Search, Table)
   ->UseManualTime()
   ->Unit(benchmark::kMillisecond)
   ->Apply(CustomArguments);
+
+void BM_contains(benchmark::State& state, bool nulls)
+{
+  const cudf::size_type column_size{(cudf::size_type)state.range(0)};
+  const cudf::size_type values_size = column_size;
+
+  auto col_data_it = cudf::detail::make_counting_transform_iterator(
+    0, [=](cudf::size_type row) { return static_cast<float>(row); });
+  auto val_data_it = cudf::detail::make_counting_transform_iterator(
+    0, [=](cudf::size_type row) { return static_cast<float>(values_size - row); });
+
+  auto column = [&]() {
+    return nulls ? cudf::test::fixed_width_column_wrapper<float>(
+                     col_data_it, col_data_it + column_size, make_validity_iter())
+                 : cudf::test::fixed_width_column_wrapper<float>(col_data_it,
+                                                                 col_data_it + column_size);
+  }();
+  auto values = [&]() {
+    return nulls ? cudf::test::fixed_width_column_wrapper<float>(
+                     val_data_it, val_data_it + values_size, make_validity_iter())
+                 : cudf::test::fixed_width_column_wrapper<float>(val_data_it,
+                                                                 val_data_it + values_size);
+  }();
+
+  for (auto _ : state) {
+    cuda_event_timer timer(state, true);
+    auto col = cudf::contains(column, values);
+  }
+}
+
+BENCHMARK_DEFINE_F(Search, ColumnContains_AllValid)(::benchmark::State& state)
+{
+  BM_column(state, false);
```

Comment:
Did you mean `BM_contains`?

Reply:
Hah, perhaps I'm not benchmarking what I intended to!

Reply:
Fixed. Now that I'm benchmarking the right thing, performance is dramatically improved!